assets/aws/files/bin/teleport-ssm-publish-tokens

```diff
@@ -26,6 +35,18 @@ KUBE_TOKEN=$(uuid)
 ${TCTL} nodes add --roles=kube --ttl=4h --token=${KUBE_TOKEN}
 aws ssm put-parameter --name /teleport/${TELEPORT_CLUSTER_NAME}/tokens/kube --region ${EC2_REGION} --type="SecureString" --value="${KUBE_TOKEN}" --overwrite
 
+# App token authenticates app services joining the cluster
+APP_TOKEN=$(uuid)
```

Comment:
hmm, why use a uuid as a join token? is it random enough?
can we generate it via `/dev/random` instead?

Reply:
This is just the way it was done everywhere else in the script. We could theoretically change to using `/dev/random` instead but i'd want to do it via a separate PR to avoid the risk of breaking something unrelated.

Reply:
I think `uuid --random` should give us a comfortable amount of entropy
the format won't change, just the kind of UUID generator

Reply:
It doesn't look like `uuid` supports that flag:

```
vagrant@linux:~$ uuid --random
uuid: invalid option -- '-'
uuid:ERROR: invalid option '-'
usage: uuid [-v version] [-m] [-n count] [-1] [-F format] [-o filename] [namespace name]
usage: uuid -d [-F format] [-o filename] [uuid]

vagrant@linux:~$ dpkg -l uuid
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version         Architecture Description
+++-==============-===============-============-===============================================
ii  uuid           1.6.2-1.5build7 amd64        Universally Unique Identifier Command-Line Tool
```

Maybe `uuid -m` would work?

```
       -m Forces the use of a random multi-cast MAC address when generating version 1 UUIDs. By default the real physical MAC address of the
          system is used.
```

Reply:
Ah, it's https://linux.die.net/man/1/uuid
Should be `uuid -v4` then